src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,88 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String COMMA = ",";
+    private static final String SUPPLY = "supply";
+    private static final String BUY = "buy";
+    private static final String RESULT = "result";
+    private static final int OPERATION_TYPE_INDEX = 0;
+    private static final int AMOUNT_INDEX = 1;
+
     public void getStatistic(String fromFileName, String toFileName) {
 
+        int[] totals = calculateTotals(fromFileName);
+
+        int totalSupply = totals[OPERATION_TYPE_INDEX];
+
+        int totalBuy = totals[AMOUNT_INDEX];
+
+        int totalAmount = totalSupply - totalBuy;
```

Comment:
```suggestion
```

replace all this logic to processReport method

Reply:
not fixed